clion/src/191/main/resources/META-INF/platform-clion-only.xml

```diff
@@ -2,4 +2,16 @@
     <extensions defaultExtensionNs="clion">
         <buildConfigurationProvider implementation="org.rust.clion.cargo.CargoBuildConfigurationProvider"/>
     </extensions>
+
+    <extensions defaultExtensionNs="com.intellij">
+        <profiler.api.attachProfilerProvider
+                implementation="com.intellij.profiler.clion.CLionLocalAttachProfilerProvider"/>
+        <programRunner implementation="org.rust.clion.profiler.RsProfilerRunner"/>
+        <executor implementation="com.intellij.profiler.clion.ProfilerExecutor"/>
```

Comment:
Hm, does it work at all?
It seems we don't need to register `attachProfilerProvider` and `executor` at all because they are already registered in CLion